enterprise/app/models/concerns/captain_tools_helpers.rb

```diff
@@ -0,0 +1,76 @@
+# Provides helper methods for working with Captain agent tools including
+# tool resolution, text parsing, and metadata retrieval.
+module CaptainToolsHelpers
```

Comment:
lets keep this under the concerns namespace ? 
would probably remove the need for us to do add the path to autoload paths